agent/src/main/java/io/jenkins/plugins/remotingkafka/Options.java

```diff
@@ -18,4 +18,13 @@
 
     @Option(name = "-secret", usage = "Secret to send to master to establish a secure connection")
     public String secret;
+
+    @Option(name = "-kafkaUsername", usage = "Username to login to kafka broker")
```

Comment:
You can make this option required via attributes. But, as suggested above, I would rather suggest supporting non-authenticated mode if the username is not set